src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect} from "react"
+import { changeVote } from "../../api"
+
+function ArticleVotes({singleArticle}) {
+    const [optimisticVotes, setOptimisticVotes] = useState("")
+    const [errorMessage, setErrorMessage] = useState("")
+
+    useEffect(() => {
+        setOptimisticVotes(singleArticle.votes)
+    }, [])
+
+    function handleVote(e) {
```

Comment:
works well so far and I like the optimistic render (and that you've undone the optimistic render too in the catch). Consider though that there is no limit to how many times I can upvote/downvote an article 0.0 (not a mandatory part of the ticket but something to think about in terms of a UX/UI perspective